### PR TITLE
Code assumes that there is a trailing slash behind all server URLs.

### LINKED ISSUE #425: "Link to Graph in Prometheus" only works when server url was configured with trailing /

### PR DESCRIPTION
**Pull request by stuartnelson3**
Enforce this slash's existence on save for server.

fixes #425 

@juliusv

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -205,4 +205,4 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
 
 BUNDLED WITH
-   1.10.1
+   1.10.4
```
Comment (stuartnelson3) [inline Gemfile.lock]: i can't believe this is part of the new bundler -.-

**Comment by juliusv** [comment]
This would add a second mechanism for slash sanitization. In all other places, we already use the `URLGenerator` to ignore the exact suffix. Can't we do the same for fixing https://github.com/prometheus/promdash/issues/425?

File: Gemfile.lock (hunk 1 above)
Comment (juliusv) [inline Gemfile.lock]: Yeah. Sheesh...

**Comment by stuartnelson3** [comment]
that's way better, updated

**Comment by juliusv** [comment]
:+1:

**Pull request merged**